projects/OG-Analytics/src/main/java/com/opengamma/analytics/financial/provider/description/interestrate/NormalSTIRFuturesProviderInterface.java

```diff
@@ -35,4 +37,10 @@
    */
   IborIndex getFuturesIndex();
 
+  /**
+   * Returns the Normal parameters.
+   * @return The parameters.
+   */
+  Surface<Double, Double, Double> getNormalParameters();
```

Comment:
This is a general interface with volatility potentially dependent of 4 parameters. There is no guarantee that the volatility will be stored as a surface. This method should probably go into the specific implementation (NormalSTIRFuturesExpSimpleMoneynessProvider and NormalSTIRFuturesExpStrikeProvider).